server/common/src/main/java/org/eclipse/lsp/cobol/common/model/tree/variable/ElementaryItemNode.java

```diff
@@ -43,7 +43,10 @@ public ElementaryItemNode(
       UsageFormat usageFormat,
       boolean redefines,
       boolean isBlankWhenZeroPresent,
-      boolean isSignClausePresent) {
```

Comment:
I'd rather better to introduce 2nd constructor with additional parameters to keep old code unchanged and to allow to define new parameters VIA new constructor

Reply:
updated